pyproject.toml

```diff
@@ -22,16 +22,16 @@ license = {file = "LICENSE"}
 requires-python = ">=3.8"
 dependencies = [
     "bokeh",
-    "mapca>=0.0.3",
+    "mapca<=0.0.4",
```

Comment:
```suggestion
    "mapca<=0.0.5",
```

Reply:
I made a new release earlier, so we could already ping to that one.

Reply:
Thanks! I actually think I should leave it as-is, since dependabot should flag it for us almost immediately after this is merged.

Reply:
Good point!